samples/bmi/calculator/src/main/java/io/servicecomb/samples/bmi/CalculatorServiceImpl.java

```diff
@@ -30,16 +38,34 @@
    * {@inheritDoc}
    */
   @Override
-  public double calculate(double height, double weight) {
+  public Map<String, String> calculate(double height, double weight) {
     if (height <= 0 || weight <= 0) {
       throw new IllegalArgumentException("Arguments must be above 0");
     }
     double heightInMeter = height / 100;
     double bmi = weight / (heightInMeter * heightInMeter);
-    return roundToOnePrecision(bmi);
+    
+	double result = roundToOnePrecision(bmi);
+
+	MicroserviceInstance name = RegistryUtils.getMicroserviceInstance();
+	String processID = name.getInstanceId().substring(0, 12);  
+	
+	Date date=new Date(); 
```

Comment:
leave space before and after the equal symbol. This will be done when you reformat the code after importing the code style.